.github/workflows/auto_label.yml

```
@@ -0,0 +1,44 @@
+name: Label PRs & Issues
+
+on:
+  issues:
+    types: [opened, edited]
+  pull_request_target:
+    types: [edited, opened, synchronize, reopened]
+
+jobs:
+  auto-label-rocm:
+    runs-on: ubuntu-18.04
+    steps:
+    - name: Retrieve information
+      id: vars
+      run: |
+        set -eux
+        IS_PR=${{ github.event.pull_request }}
+        if [[ -n "${IS_PR}" ]]; then
```

This copies the entire payload into the `IS_PR` variable, right? Would it maybe be more straightforward to do something like this instead?
```suggestion
        if [[ "${{ github.event_name }}" == "pull_request_target" ]]; then
```
Although actually, now that I think about it, given that you're using subfields of `pull_request` later, the way you're currently doing it might be more clear.